OOP/assignment2/SurveyApplication.java

```diff
@@ -0,0 +1,204 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SurveyApplication {
+	private static ArrayList<Question> questionList = new ArrayList<Question>();
+	private static ArrayList<Particpant> participantList = new ArrayList<Particpant>();
+
+	public static void main(String args[]) {
```

Comment:
ACTION: Modular implementation needed. You main class should contain minimum no. of code and hardly any logical/function implementation should be there. E.g. In this you can only instantiate "Persons" taking the survey and they all call a method in which actual logic is implemented. Currently your code is completely dependent on the "main" method and as I have already told you the main method should be decoupled from the solution and only minimum things (which you would use in test methods) be there.